packages/plugin-debridge/src/__tests__/mocks/handlers.ts

```diff
@@ -0,0 +1,170 @@
+import { http, HttpResponse } from 'msw';
```

Comment:
I think this is misleading for your passing tests; by mocking the responses, we're not testing against the actual provider. 

Review and merge my update PR, then run tests. You will see that we're definitely hitting the API correctly and getting some good data, but I think we're getting some bad requests too:

Failed to get liquidity for route: { error: 'Request failed after 2 attempts: HTTP 400: Bad Request' }